CHANGELOG.d/feature_is_reflectable.md

```diff
@@ -0,0 +1,6 @@
+* Implement the IsReflectable type class
+
+  The `IsReflectable` type class is a common interface for reflecting
+  type-level values down to the term-level. Its instances are
+  automatically solved by the compiler, and it allows `Symbol` and `Int`
+  kinded types to be reflected to their term-level representations.
```

Comment:
Ah, shoot, not quite. This needs rewording to account for the rename and the new instances.

```suggestion
* Implement the Reflectable type class

  The `Reflectable` type class is a common interface for reflecting
  type-level values down to the term-level. Its instances are
  automatically solved by the compiler, and it allows `Symbol`, `Int`,
  `Bool`, and `Ordering` kinded types to be reflected to their term-level
  representations.
```

Reply:
If this is all that needs to be changed, can't we just commit the suggestion and merge? Or is there potentially more in the changelog that needs to be edited?

Reply:
> If this is all that needs to be changed, can't we just commit the suggestion and merge?

I think we should give the original authors plenty of time to respond. I don't think we need to be in a hurry about this.

Reply:
You're right.